app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require_relative 'lib/database'
+require 'dotenv'
+
+Dotenv.load
+database = Database.new
+
+get '/' do
+  kitten = database.random_row('animals', "type = 'kitten'")
+  @kitten_url = m_url(kitten[:link])
+  @kitten_id = kitten[:id]
+  puppy = database.random_row('animals',"type = 'puppy'")
+  @puppy_url = m_url(puppy[:link])
+  @puppy_id = puppy[:id]
+  @kitten_percent = database.kitten_percent
+  erb :index
+end
+
+get '/vote/:animal_id' do
+  database.vote(params['animal_id'])
+  redirect to('/')
+end
+
+def m_url(link)
+  splitted = link.split(".")
+  splitted[-2]<<"m"
+  splitted.join(".")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.